File: src/src/gestorAplicacion/libreria/EjemplarLibro.java
package gestorAplicacion.libreria;

import java.io.Serializable;

public class EjemplarLibro extends Ejemplar implements Serializable {
    Libro libro;

    public EjemplarLibro(int id, EstadoEjemplar estadoEjemplar, Libro libro) {
        super(id, estadoEjemplar);
        this.libro = libro;
    }

    public Libro getLibro() {
        return libro;
    }
}

File: Python/gestorAplicacion/libreria/ejemplarLibro.py

from Python.gestorAplicacion.libreria.ejemplar import Ejemplar


class EjemplarLibro(Ejemplar):
    def __init__(self, identificador: int, estadoEjemplar, libro):
        super(EjemplarLibro, self).__init__(identificador, estadoEjemplar)
        self._identificador = identificador
        self._estadoEjemplar = estadoEjemplar
        self._libro = libro

    def getLibro(self):
        return self._libro